### PR TITLE
Doc strings

### PR DESCRIPTION
**Pull request by sschachten**
- added missing docstrings for functions, classes and scripts

- updated some scripts to the new line length of 160 characters

- added some additional enumerations

### CODE REVIEW

File: materialdatabase/material_data_base_classes.py [hunk 1]
```diff
@@ -448,18 +394,18 @@ def compare_core_loss_flux_density_data(self, matplotlib_widget, material_list:
...
-            @param matplotlib_widget: plot parameter for GUI
-            @param material_list:[material1, material2, ....]
-            @param flux_density_list: [flux1, flux2,..]
-            @return: return plotting data in two list, power_loss and temperature
-            """
+        Compare the core loss of a material at different magnetic flux densities over the temperature from datasheet.
+
+        @param matplotlib_widget: plot parameter for GUI
+        @param material_list:[material1, material2, ....]
+        @param flux_density_list: [flux1, flux2,..]
```
Comment (gituser789) [inline materialdatabase/material_data_base_classes.py:401]: must be :param 
@param is another style guide, but we are using :param

File: materialdatabase/material_data_base_classes.py [hunk 2]
```diff
@@ -488,21 +433,20 @@ def compare_core_loss_temperature(self, matplotlib_widget, material_list: list,
...
 
-    def compare_core_loss_frequency(self, matplotlib_widget, material_list: list, temperature_list: list = None,
-                                    flux_density_list: list = None):
+    def compare_core_loss_frequency(self, matplotlib_widget, material_list: list, temperature_list: list = None, flux_density_list: list = None):
+        """
+        Compare the core loss of a material at different temperatures over the frequency from datasheet.
+
+        @param matplotlib_widget: plot parameter for GUI
+        @param material_list:[material1, material2, ....]
+        @param flux_density_list: [flux1, flux2, ....]
```
Comment (gituser789) [inline materialdatabase/material_data_base_classes.py:442]: :param

File: materialdatabase/material_data_base_classes.py [hunk 3]
```diff
@@ -568,27 +510,26 @@ def compare_b_h_curve(self, matplotlib_widget, material_list: list, temperature_
...
-    def compare_permeability_measurement_data(self, matplotlib_widget, material_list: list, measurement_name: list,
-                                              frequency_list: list = None,
+    def compare_permeability_measurement_data(self, matplotlib_widget, material_list: list, measurement_name: list, frequency_list: list = None,
                                               temperature_list: list = None, plot_real_part: bool = False):
         """
-            Method is used to compare material properties from measurement data.
-            @param matplotlib_widget: plotting parameter for GUI
-            @param material_list:[material1, material2, .....]
-            @param plot_real_part: True for plot real part of mu/ False for plots imaginary part of mu
-            @type temperature_list: [temp1, temp2, ...]
```
Comment (gituser789) [inline materialdatabase/material_data_base_classes.py:579]: :param

**Review by gituser789 — CHANGES_REQUESTED** [review]

File: materialdatabase/material_data_base_classes.py (hunk 2 above)
Comment (sschachten) [reply materialdatabase/material_data_base_classes.py:442]: Changed

**Review by gituser789 — APPROVED** [review]

**Pull request merged**